src/Driver/FlatFileDriver.php

```diff
@@ -25,13 +25,13 @@ public function __construct($baseDirectory)
      */
     public function listQueues()
     {
-        $it = new \DirectoryIterator($this->baseDirectory, \FilesystemIterator::SKIP_DOTS);
+        $it = new \DirectoryIterator($this->baseDirectory);
 
         $queues = [];
 
         foreach ($it as $file) {
             /** @var $file \SplFileInfo */
-            if (!$file->isDir()) {
+            if (!$file->isDir() || in_array($file->getBaseName(), array('.', '..'))) {
```

Comment:
When you iterate over a DirectoryIterator it will return other DirectoryIterator instances. The inlined phpdoc is wrong and causes confusion (another reason why i dont like them).

http://php.net/manual/en/directoryiterator.isdot.php